other-docs/guides/updating-php/README.md

```diff
@@ -9,9 +9,14 @@ There are 2 key steps to getting ready for a new version of PHP:
 
 ## Altis Compatibility Chart
 
+**Note:** PHP 8.0 will officially be available from the Altis v12 release date onwards. Altis v11 will run on PHP 8.0 but support is experimental. Keep reading to learn how to prepare your application for a new PHP version.
+
 |Altis|PHP Version|
 |-|-|
-|v7+|7.4|
+|v13|8.0|
+|v12|7.4-8.0|
+|v11|7.4-8.0 (experimental)|
```

Comment:
do we think ( might be over thinking this ) that it would be better to say`7.4, 8.0(experimental)`? 

Don't want people to think 7.4 is experimental. 😁

Reply:
Hopefully it's clear to devs but we could change the hyphenated ranges to comma separated as you suggest, it's not like there's actually enough different versions to warrant using a range, it's literally 7.4 and 8.0.

I agree I think we should do this, change the hyphens to commas.